wri/gfw
# feat(funders-logo): add new logo for International Climate Initiative

## Overview

Add new Logo to Funders Section

### Acceptance criteria

- [x] The attached logo is added to the “Funder” section of the about page
- [x] To follow the current alphabetical listing, this will be between the IDB invest and Macarther Foundation
- [x] Just like other logos on this page, there is a link attached to this image, https://www.international-climate-initiative.com/en  , which will open to a new tab
- [x] This should be added before June 30

## Demo

<img width="1201" alt="Screenshot 2023-06-23 at 12 51 17" src="https://github.com/wri/gfw/assets/23243868/47e88ded-f7bc-40b8-8c69-787f90aae447">


## Testing

- Open "About Page"
- Check new logo
